src/net.cpp

```diff
@@ -458,13 +458,30 @@ bool CNode::IsBanned(CNetAddr ip)
     return fResult;
 }
 
-bool CNode::Ban(const CNetAddr &addr) {
+bool CNode::Ban(const CNetAddr &addr, int64_t bantimeoffset) {
```

Comment:
Right. I didn't want to change this in this PR and i kept it for legacy reasons and to lower the risk of breaking something.